jscomp/core/lam.ml

```diff
@@ -92,7 +95,7 @@ module Types = struct
     | Llet of Lam_compat.let_kind * ident * t * t
     | Lletrec of (ident * t) list * t
     | Lprim of prim_info
-    | Lswitch of t * switch
+    | Lswitch of t * lambda_switch
```

Comment:
Any particular reason to change the name here?

Reply:
for consistency with module Lambda (no need to remember 2 names for the same thing), but it makes little difference